tools/tests/package-tests.js

```diff
@@ -1952,3 +1958,637 @@ selftest.define("show package w/many versions",
     ]
   });
  });
+
+
+// This tests that we get the right excerpt out of the Readme.md in different
+// combinations. It doesn't test publication, because publishing is slow --
+// that's covered in a different test.
```

Comment:
Ah, cool, that used to not be the case! Good to know!

Though I am not sure that actually testing the full process here is the wrong approach. It is certainly something that we *can* test! It also tests that we render things properly, which is useful, especially if the mechanics of what we do about it (processing vs not processing markdown, for example) ever change.